lib/src/models/seed.dart

```diff
@@ -25,9 +25,7 @@ class Seed extends Equatable {
   /// Name of seed itself, this name stores in app's storage, not in keystore
   final String? _name;
 
-  // TODO(alex-a4): replace masterKey.key.name to masterKey.key.toEllipse()
-  /// If seed has name, it will be returned, otherwise master key name.
-  String get name => _name ?? masterKey.key.name;
```

Comment:
Comment `If seed has name, it will be returned, otherwise master key name` is necessary :)

Reply:
But I think, it should be corrected to contains info about `publickey.toEllipse`